kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -503,6 +503,7 @@ private String getFieldType(FormField field) {
              default: 
                  switch (field.getType()) {
                      case "java.time.LocalDateTime":
+                         return "datetime-local";
```

Comment:
This does not change the result of the switch, is that a leftover?

Reply:
it's a short name and will be used in the HTML template.
https://github.com/kiegroup/droolsjbpm-integration/pull/2768/files#diff-f490d20f356970cad0c84cd0ffe2f5396127ad65bcf5a6d08c0ee3439efbb4f9R65

Reply:
There's no operational difference in previous:
```
                     case "java.time.LocalDateTime":
                     case "java.util.Date":
                     case "java.sql.Date":
                     case "java.sql.Timestamp":
                         return "datetime-local";
```
and after this change:
```
                     case "java.time.LocalDateTime":
                         return "datetime-local";
                     case "java.util.Date":
                     case "java.sql.Date":
                     case "java.sql.Timestamp":
                         return "datetime-local";
```
So IMO this should be fixed or reverted.

Reply:
previous "java.time.LocalDateTime" return default name, now is "datetime-local"

Reply:
In original switch the case statement `"case "java.time.LocalDateTime":` was activated when input matching the literal so the first `return` value was actually returned. That's the way of execution in case statement **without break statement** is to continue in the following lines (and also cross cases) until it reaches interruption point (usually a `break` statement in this case it's `return`). In any case those two switches are identical when it comes to how it's executed.
The `default` branch is activated only when none of the cases matched the input value.

Reply:
https://stackoverflow.com/questions/18192255/regarding-java-switch-statements-using-return-and-omitting-breaks-in-each-case

Reply:
@pefernan wdyt, should we rollback this change? IMO it's unnecessary isolated change of coding style - there is another approach applied for the rest of cases, so diverging might puzzle people (it did puzzle myself).

Reply:
totally